website/src/pages/roadmap.md

```diff
@@ -10,48 +10,51 @@ down by areas on our [stack](blog/2021/07/21/streaming-data-lake-platform/#hudi-
+| Efficient bootstrap and migration of existing non-Hudi dataset | 0.14.0 onward   | [HUDI-1265](https://issues.apache.org/jira/browse/HUDI-1265)                                                                                                                                                                                             |
 | Lock free concurrency control                                  | 1.0.0 onward   | [HUDI-3187](https://issues.apache.org/jira/browse/HUDI-3187)                                                                                                                                                                                             |
 | Non-blocking/Lock-free updates during clustering               | 1.0.0 onward   | [HUDI-1042](https://issues.apache.org/jira/browse/HUDI-1042)                                                                                                                                                                                             |
 | Time Travel updates, deletes                                   | 1.0.0 onward   ||
 | General purpose support for multi-table transactions           | 1.0.0 onward   ||
+| Lockless multi-writer support                                  | 1.0.0          | [RFC-66](https://github.com/apache/hudi/pull/7907)                                                                                                                                                                                                        |
+| A more effective HoodieMergeHandler for COW table with parque  | 1.0.0          | [RFC-68](https://github.com/apache/hudi/blob/f1afb1bf04abdc94a26d61dc302f36ec2bbeb15b/rfc/rfc-68/rfc-68.md)                                                                                                                                               |
+
 
 ## Programming APIs
 
 |Feature| Target Release |Tracking|
 |------------|----------------|-----------|
-| Redesign and optimization of record payload abstraction | 0.13.0         | [RFC-46](https://github.com/apache/hudi/blob/master/rfc/rfc-46/rfc-46.md), [HUDI-3217](https://issues.apache.org/jira/browse/HUDI-3217) |
-| Optimized storage layout for cloud object stores | 0.13.0         | [RFC-60](https://github.com/apache/hudi/pull/5113), [HUDI-3625](https://issues.apache.org/jira/browse/HUDI-3625) |
-| Support of verification with multiple event_time fields | 1.0.0 onward   | [RFC-59](https://github.com/apache/hudi/pull/6382), [HUDI-4569](https://issues.apache.org/jira/browse/HUDI-4569) |
+| New Hudi Table Spec API for Query Integrations                | 1.0.0         | [RFC-64](https://github.com/apache/hudi/pull/7080)                                                              |
```

Comment:
looks like we don't have anything on this title for 0.14.0. 
@yihua @codope @xushiyan  : can we think of something under this title for 014.0 ?

Reply:
We landed this in 0.14.0: MOR snapshot reader to integrate with query engines without using Hadoop APIs https://github.com/apache/hudi/pull/9066 .  Is it worth mentioning?